app/component/itinerary/ItineraryPage.js

```diff
@@ -1187,7 +1224,10 @@ export default function ItineraryPage(props, context) {
       bikePlan?.edges?.length ||
       bikePublicPlan?.edges?.length ||
       parkRidePlan?.edges?.length ||
-      (settings.includeCarSuggestions && carPlan?.edges?.length));
+      (settings.includeCarSuggestions &&
+        (carPlan?.edges?.length ||
+          (config.carBoardingModes?.FERRY !== undefined &&
+            carPublicPlan?.edges?.length))));
```

Comment:
(settings.includeCarSuggestions && carPlan?.edges?.length) ||
 carPublicPlan?.edges?.length

Reply:
This assumes that the carpublicplan would be shown even without the setting. I think this is wrong. Do you agree? FERRY is now removed

Reply:
Combination (settings.includeCarSuggestions && carPlan?.edges?.length) is used because carplan may be comouted for co2 comparison only.

Reply:
This has now been changed to (settings.includeCarSuggestions && carPlan?.edges?.length) ||
carPublicPlan?.edges?.length > 0. I had a double check here and forgot I checked this earlier